### DISCUSSION TITLE
Altruistic tower list

### DISCUSSION THREAD

**Discussion by sr-gi** [talaia-labs/rust-teos#158] [Show and tell]
Let's use this to share our altruistic tower details in the following format so people can benefit from it!

`<tower-id>@<host>:<port>`

I'll start:

**Clearnet (same tower, two hostnames)** 

- `02b4891f562c8b80571ddd2eeea48530471c30766295e1c78556ae4c4422d24436@teos.talaia.watch:9814`
- `02b4891f562c8b80571ddd2eeea48530471c30766295e1c78556ae4c4422d24436@mainnet.eyeofsatoshi.watch:9814`

**Tor** `02b4891f562c8b80571ddd2eeea48530471c30766295e1c78556ae4c4422d24436@recnedb7xfhzjdrcgxongzli3a6qyrv5jwgowoho3v5g3rwk7kkglrid.onion:9814`

PS: This is inspired by @openoms [altruistic tower list](https://github.com/openoms/lightning-node-management/issues/4) (for LND)

**Comment by decentralizedb** [comment]
Mine under Tor

Tor
`03488e12bc580caaaa4f575bde2b4754746382fc5e8b5e3d03a632b0f15c5175cc@nbwvzyiicsg7o4yoilzg3t3uoxe6pzby2gb4t45boyzg2okyx7z6ppid.onion:9814`

**Comment by Capitan-Man** [comment]
Under tor:

`022233128e4ca07e29afd13ee4bb94c9fa172e7f8b9452e6acf528cc53dc2b2b26@ykquhk73nhcjht4fdphnkshvwjeexqrpn2marp22yyvdnwxxdeaps5ad.onion:2121`

**Comment by jochemin** [comment]
**Tor**
`028b85eb14c3d571da4aeac17ec2bacc251a9356180a9b96c8bc1198a74a799eb4@zn55t2ihb5penb4bmygmho6w25xpdgprrz3mg64rcgiaz2v6hdx2gfqd.onion:2121`

**Comment by lunaticoin** [comment]
**Tor**
`029ba52e776f450fbc10ac237c84579ca001a94945fd93881b0657e23c8df0b1af@ygrb66xi53wjb52bwdy7y5dyazandyr3qwctz4cceimq4sqaoignbbyd.onion:2121`

**Reply by svpi11** [reply]
yours seems not reachable :)

**Comment by mariocynicys** [comment]
Tor
`031421fe4a6c5b0b2197443b2b347dbafe132ae18c9e0dcb0b284123f1ea6ffe40@kfve5m2gzkreovtfljef256aolpwmz4zwooequ4ipa32kgpannhjieid.onion:9814`

Clear net
`031421fe4a6c5b0b2197443b2b347dbafe132ae18c9e0dcb0b284123f1ea6ffe40@teos.mariocynicys.me:9814`

**Reply by mariocynicys** [reply]
The tower was down in the height range: 793550-798452 due to memory issues.
It's back online now.

**Comment by 3cola** [comment]
Tor
`03cb971ac01dbab0516e02b34cab1f3fe48b654ccef7345665e4e725a6806e3532@tmt7k3fta6w6ra73sgvgrn23lhsaf722kf36llnzg5prqefjjcrzrxid.onion:2121`

**Comment by ulrichard** [comment]
Clearnet:

`0212461401dd0a8016a02aa87dd02ad07c86d415885da3b3e4445ba975cadbb4f4@ulrichard.ch:9814`

`027f4e064c2b15653b6cd36982656db0662822a5695b0fc5e86a570df159b70723@65.109.24.172:9814`

`03408b59231eda297d3bcdd15afb729b05c1a360d471df893728291b1252a33b1e@65.108.101.79:9814`

**Comment by sat-hoarder** [comment]
**Clearnet:**
`03b0a47228c8b36477002d745cf578a4db2b7c2ba2e2aee2c65419053e4375a1dd@satseye.slicksparks.ky:9814`

**Tor:**
`03b0a47228c8b36477002d745cf578a4db2b7c2ba2e2aee2c65419053e4375a1dd@ekeyon2bbc3bmixlid2c6qgo5rapuilknz7sr4h36wwt2j2ockag24yd.onion:2121`

**Comment by fbrz76** [comment]
Clearnet, paid, but 24/7 powerfull service.
[frznode TEOS service](https://microlancer.io/service/view/1413)

Tor:
`032a4ff7dd37ae01de848c3029c91253e0f30ee9038dcbcd08d6338bbbcb11339a@devrzfhfi2xay3ur5fpiualkqq6sedkaykvwsaf4in5rprohsoxfqoad.onion:9814`

**Comment by orbitalturtle** [comment]
Tor: 
`03d8e71945e7245e9a63ae4c3bc7b6cd020adf808706237cf95fefa394e7b2d180@dbed7lompa5u7y5eubhcfrnknpplh35o4vqh7zjfl7pypd7szvhlihyd.onion:9814`
